api/Clients.json

```diff
@@ -133,20 +133,27 @@
             "firefox": [
               {
                 "version_added": "54",
-                "notes": "<code>Client</code> objects returned in most recent focus order."
+                "notes": "<a href='https://www.mozilla.org/en-US/firefox/organizations/'>Extended Support Releases (ESR)</a> before Firefox 78 ESR do not support service workers and the Push API."
```

Comment:
I don't love duplicating this note, but that's what we have to do when the version ranges for two notes overlap partially. I've sent https://github.com/mdn/browser-compat-data/pull/17060 to delete these notes in particular, which should then make this cleanup cleaner.